wordpress-mobile/WordPress-Android
# Support fragment migration: Media

In this PR we :

- deleted `MedaEditFragment`, which apparently wasn't being used
- made `MediaGridFragment` be a `support.v4.Fragment`

To test:
1. go to the Media section in the app
2. verify it works (i.e. you can browse media of your site)
3. do rotate phone and verify it works
